### ISSUE TITLE
[Feature] Make it possible to change the default expect(...) timeout

### ISSUE THREAD

**Issue by DavidK-Visiba** [microsoft/playwright-python#1989]
I need to change the default timeout value for the `expect(...)` statements in my Python code from 5000ms to something else.

In the Node.js version, it is possible to set a default timeout for the expect() statements. I can't find any corresponding function in the Python version, neither in the sync api nor in the async api.

Ref doc for Node.js: https://playwright.dev/docs/test-timeouts#expect-timeout

This page is not available for the Python version (Switch from Node.js to Python in the drop-down in the menu bar), so I guess the functionality is missing.

Basically, I guess I need a Python equivalent to this piece of Node.js code:

```
import { defineConfig } from '@playwright/test';

export default defineConfig({
  expect: {
    timeout: 10 * 1000,
  },
});
```

**Comment by mxschmitt** [comment]
It's on a different page for Python see here: https://playwright.dev/python/docs/test-assertions#setting-a-custom-timeout

**Comment by DavidK-Visiba** [comment]
Thank you, @mxschmitt! 👍
